python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -305,8 +308,6 @@ class GrapheneAssetNode(graphene.ObjectType):
     hasAssetChecks = graphene.NonNull(graphene.Boolean)
     assetChecksOrError = graphene.Field(
         graphene.NonNull(GrapheneAssetChecksOrError),
-        limit=graphene.Argument(graphene.Int),
-        pipeline=graphene.Argument(GraphenePipelineSelector),
```

Comment:
what's your confidence level that nobody is currently progamatically querying this?

Reply:
some discussion about the pipeline filter here: https://dagsterlabs.slack.com/archives/C05KFG9TETB/p1699888783117649